endpoints.json

```diff
@@ -25,5 +38,38 @@
         }
       ]
     }
+  },
```

Comment:
You should add on to this file as you go along the tasks, not do it all from the beginning. This is meant to be an overview of everything existing within your API, currently you don't have these yet!